integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/DashboardWomenSpecification.groovy

```diff
@@ -0,0 +1,345 @@
+package uk.gov.justice.digital.hmpps.cattool.specs
+
+import groovy.json.JsonOutput
+import uk.gov.justice.digital.hmpps.cattool.pages.DashboardInitialPage
+import uk.gov.justice.digital.hmpps.cattool.pages.DashboardRecatPage
+import uk.gov.justice.digital.hmpps.cattool.pages.SupervisorHomePage
+
+import static uk.gov.justice.digital.hmpps.cattool.model.UserAccount.SUPERVISOR_USER
+import static uk.gov.justice.digital.hmpps.cattool.model.UserAccount.SUPERVISOR_USER
+import static uk.gov.justice.digital.hmpps.cattool.model.UserAccount.WOMEN_SUPERVISOR_USER
+
+class DashboardWomenSpecification extends AbstractSpecification {
+
+  enum CatType {
+    INITIAL, RECAT
+  }
+
+  enum SecurityType {
+    MANUAL, AUTO, FLAGGED
+  }
+
+  class JasonBuilder {
+    CatType catType
+    String suggested
+    String overridden
+    String supervisor
+    SecurityType securityType
+
+    JasonBuilder securityType(securityType) {
+      this.securityType = securityType
+      return this
+    }
+
+    String build() {
+      def sec = securityType == SecurityType.MANUAL ? [securityInputNeeded: 'Yes'] : null
+      def contents
+      if (catType == CatType.INITIAL) {
+        contents = [categoriser: [provisionalCategory: [suggestedCategory: suggested, overriddenCategory: overridden]],
+                    supervisor : [review: supervisor ? [supervisorOverriddenCategory: supervisor] : null],
+                    ratings    : [securityInput: sec]]
+      } else {
+        contents = [recat     : [decision: [category: suggested], securityInput: sec],
+                    supervisor: [review: supervisor ? [supervisorOverriddenCategory: supervisor] : null],
+        ]
+      }
+      return JsonOutput.toJson(contents)
+    }
+  }
+
+  static final SECURITY_AUTO = '{"socProfile":{"transferToSecurity":"true"}}'
+
+  /** convenience method to make data setup calls as succinct as possible */
+  def dbRow(bookingId, prisonId, catType, startDate, referredDate, securityReviewedDate, assessmentDate, approvalDate, dueByDate, json, riskProfile = '{}') {
+    db.doCreateCompleteRow(-bookingId, bookingId, json, catType == 'INITIAL' ? 'FEMALE_USER' : 'FEMALE_USER', 'APPROVED', catType, null, referredDate, null, 1, riskProfile,
+      prisonId, "ON${bookingId}", startDate, null, securityReviewedDate, approvalDate, assessmentDate, dueByDate)
+  }
+
+  int dbCatRow(bookingId, seq, approvedDate, prisonId, suggested, supervisor = null, catType = 'RECAT') {
+    return db.doCreateCompleteRow(-bookingId * seq, bookingId, rc(suggested, supervisor).build(), 'FEMALE_USER', 'APPROVED', catType, null, null, null, seq, '{}',
```

Comment:
Is this `-bookingId * seq` and `-bookingId` above correct?  Just thinking about the perspective of the db cleanup actions. If it ends up as a positive number, is there a chance that it is > 1000 and clearDb() not picking it up?